compiler/quilt/tools/package.py

```diff
@@ -365,7 +373,9 @@ def _add_to_contents(self, fullname, hashes, ext, path, target, fmt):
 
         try:
             target_type = TargetType(target)
-            if target_type is TargetType.PANDAS:
+            if target_type is TargetType.GROUP:
+                node = GroupNode(dict())
```

Comment:
This feels hacky to me... The code above - `ptr = ptr.children.setdefault(...)` - already handles the creation of groups.

Reply:
The alternative is to special case `ipath.pop()` above so either way we will require a dedicated codepath for `GROUP` vs `FILE`. It's also nice to keep the last assignment in the routine as is (and not special-case `ipath.pop()`). We could signal `GROUP` with `target=None` but, given that we have (and need) `save_group`, it's probably better to be explicit about Group creation.